### PR TITLE
Update the build steps to support ORT on Jetson

### PR DESCRIPTION
**Pull request by chilo-ms**
**Description**: Update the build steps to support ORT on Jetson

**Motivation and Context**
- Why is this change required? What problem does it solve?
PM suggests to update the BUILD.md to make it clear.

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -191,9 +191,32 @@ See more information on the TensorRT Execution Provider [here](./docs/execution_
 
 Dockerfile instructions are available [here](./dockerfiles#tensorrt)
 
-#### Jetson (ARM64 Builds)
+#### Jetson TX1/TX2/Nano(ARM64 Builds)
 
-See [instructions](https://github.com/microsoft/onnxruntime/issues/2684#issuecomment-568548387) for additional information and tips related to building Onnxruntime with TensorRT Execution Provider on Jetson platforms (TX1/TX2, Nano)
+
+1. ONNX Runtime v1.2.0 or higher requires TensorRT 7 support, make sure Jetson device is upgraded to [JetPack](https://docs.nvidia.com/jetson/jetpack/release-notes/) 4.4 or higher.
```
Comment (jywu-msft) [inline BUILD.md:197]: were you able to validate this?
JetPack 4.4 looks to be in preview and seems to depend on TensorRT 7.1 and cuDNN 8.0
AFAIK, we've only been testing TensorRT 7.0 , cuDNN 7.6

Comment (chilo-ms) [reply BUILD.md:197]: I tried to upgrade the TX2 device to JetPack 4.4 from JetPack 4.3 thorugh sdkmanager, and encountered many errors. For examples, "Unhandled error when running sudo apt-get -y update" and "dpkg install on target failed". It's possible that JetPack 4.4 is in preview and not stable yet as you mentioned. I will keep following it.

At the same time, I've successfully built and run onnxruntime v1.1.2 with JetPack 4.3 which uses TenssorRT 6. I will revise the doc to describe how to build older onnxruntime with TenssorRT 6 support.

Comment (jywu-msft) [reply BUILD.md:197]: Thanks! In addition to the issues you encountered, there will probably need to be some code changes required for ORT to support TensorRT 7.1 properly. 

Btw, according to Nvidia, skipping TensorRT 7.0 for JetPack was a rare exception.
For now, we'll have to go with documenting TensorRT 6 using Onnxruntime 1.1.2 as you suggested.  
Thanks.

Comment (chilo-ms) [reply BUILD.md]: Got it and I just made an update.

**Comment by jywu-msft** [comment]
@hariharans29 , did we get a resolution on how to merge doc-only changes? some of these pipelines are configured not to trigger for doc updates. see above comment where I tried to trigger MacOS builds.
I approved the PR but we cannot merge it.
This behavior is different than the PR I had pointed to you from months ago. https://github.com/microsoft/onnxruntime/pull/2730

**Pull request merged**